commands/saml.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+var SamlCmd = &cobra.Command{
+	Use:   "saml",
+	Short: "SAML related utilities",
+}
+
+var SamlAuthDataReset = &cobra.Command{
+	Use:     "authdatareset",
+	Short:   "Reset AuthData field to Email",
+	Long:    "Resets the AuthData field for SAML users to their email. Run this utility after setting the 'id' SAML attribute to an empty value.",
+	Example: " saml authdatamigrate",
+	RunE:    withClient(samlAuthDataResetCmdF),
+}
+
+func init() {
+	SamlAuthDataReset.Flags().Bool("include-deleted", false, "Include deleted users")
+	SamlAuthDataReset.Flags().Bool("dry-run", false, "Dry run only")
+	SamlAuthDataReset.Flags().StringSlice("users", nil, "Comma-separated list of user IDs to which the operation will be applied")
```

Comment:
If this flag is required, we can use something like `SamlAuthDataReset.MarkFlagRequired("users")` here, tho I don't know if this works with `StringSlice`. If it doesn't, we can check that `len(userIDs) > 0` inside the command

Reply:
The flag is not required. The backend will handle the cases where userIDs is null or empty.

Reply:
I see, makes sense, then I think this is a good case for using the `examples` to let the user know what will happen if the flag is not set :)